### PR TITLE
create react routing

### PR DESCRIPTION
**Pull request by ValeryiaMIRON**
- Added routing in app
https://trello.com/c/tlRODTyT/43-fe-create-react-router

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,8 +1,31 @@
 import './App.scss';
 import React from 'react';
+import { BrowserRouter as Router, useLocation, Route, Switch } from 'react-router-dom';
+// import { Route, Switch } from 'react-router';
```
Comment (nsukhotskaya) [inline src/App.jsx:4]: Why do you keep that comment ?

File: src/App.jsx [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+    </div>
+  );
+};
 
 function App() {
-  return <div className="App" />;
+  return (
+    <Router> 
+      <div className="App">
+        <PagesAll />
```
Comment (nsukhotskaya) [inline src/App.jsx:23]: You can use Switch and Route components inside App without extra function PagesAll

File: src/App.jsx [hunk 3]
```diff
@@ -1,8 +1,21 @@
 import './App.scss';
 import React from 'react';
+import { BrowserRouter as Router, useLocation, Route, Switch } from 'react-router-dom';
 
 function App() {
-  return <div className="App" />;
+  const location = useLocation();
+  return (
+    <Router> 
+      <div className="App">
```
Comment (nsukhotskaya) [inline src/App.jsx:9]: I think we don't need anymore div tag.

Comment (ValeryiaMIRON) [reply src/App.jsx]: done

**Pull request merged**